kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js

```diff
@@ -31,6 +31,11 @@ const coachStrings = createTranslator('CommonCoachStrings', {
     context:
       "In the Plan > Quizzes section, coaches can create new quizzes using the 'New quiz' button.",
   },
+  plan: {
```

Comment:
This looks like a new string to me? Breaking string freeze.

Reply:
Ah, good catch. This is already in commonCoach from before string freeze and this is a duplicate that I didn't intent to re-add with this commit.